lang/LangSource/SC_LanguageClient.cpp

```diff
@@ -99,8 +99,9 @@ void SC_LanguageClient::initRuntime(const Options& opt) {
     // start virtual machine
     if (!mHiddenClient->mRunning) {
         mHiddenClient->mRunning = true;
-        if (opt.mRuntimeDir) {
-            int err = chdir(opt.mRuntimeDir);
+        if (opt.mRuntimeDir.data() != nullptr) {
```

Comment:
If the string is empty, the return value of `data()` is undefined! I guess you meant to write
`if (!opt.mRuntimeDir.empty()) {`